server/app.py

```diff
@@ -118,8 +118,17 @@ def get_stop_predictions(stop_ids, operator, stop_name):
         all_incoming_buses = data.get('ServiceDelivery', {}).get('StopMonitoringDelivery', {}).get('MonitoredStopVisit')
         for bus in all_incoming_buses:
             route_name = bus.get('MonitoredVehicleJourney', {}).get('LineRef')
-            route_destination = bus.get('MonitoredVehicleJourney', {}).get('MonitoredCall', {}).get('DestinationDisplay').title()
-            expected_arrival = bus.get('MonitoredVehicleJourney', {}).get('MonitoredCall', {}).get('AimedArrivalTime')
+            monitored_call = bus.get('MonitoredVehicleJourney', {}).get('MonitoredCall', {})
+
+            route_destination = monitored_call.get('DestinationDisplay')
+
+            # handle potential null destination error
+            if route_destination is not None:
+                route_destination = route_destination.title()
+            else:
+                route_destination = "Unknown Destination"
+
+            expected_arrival = monitored_call.get('AimedArrivalTime')
```

Comment:
couldnt we use a string in the default value instead
```suggestion
            route_destination = bus.get('MonitoredVehicleJourney', {}).get('MonitoredCall', {}).get('DestinationDisplay', 'Unknown Destination').title()
            expected_arrival = bus.get('MonitoredVehicleJourney', {}).get('MonitoredCall', {}).get('AimedArrivalTime')
```